beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -44,7 +44,9 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 	var lastEmptyRequests int
 	errChan := make(chan error)
 	// Step 1 - Sync to end of finalized epoch.
-	for s.chain.HeadSlot() < helpers.StartSlot(highestFinalizedEpoch()+1) {
+	for s.chain.HeadSlot() < helpers.StartSlot(highestFinalizedEpoch()) {
+		log.WithField("head status:", s.chain.HeadSlot()).Debugf("helpers.StartSlot(highestFinalizedEpoch()+1)-1: %v", helpers.StartSlot(highestFinalizedEpoch()))
```

Comment:
i would not put this as normal debug log

Reply:
removed the log